### PR TITLE
Fix/integrate section

### PR DESCRIPTION
**Pull request by mnedelchev-vn**
Hi @m4sterbunny @sukanyaparashar @0xlucian ,
this PR includes Docs changes inside **Integrate** section and more specifically **WalletConnect**, **MetaMask** and **Chainlink**. I haven't changed anything into **The Graph** & **Pyth** pages, because I made a task for the changes there to @m4sterbunny and by the time I made it I had no idea that we will be updating the whole documentation soon. lmk if you see smth wrong, thanks

### CODE REVIEW

**Comment by mnedelchev-vn** [comment]
@m4sterbunny please don't miss this one, thank you!

File: docs/developing/integrate/wallets/integrating_walletconnect.mdx [hunk 1]
```diff
@@ -63,84 +65,81 @@ For this guide, we are configuring a React App.
...
- ```
+const chains = [neonMainnet, neonDevnet];
+const wagmiConfig = defaultWagmiConfig({chains, projectId, metadata});
 
-</TabItem>
-</Tabs>
+const modal = createWeb3Modal({wagmiConfig, projectId, chains});
+```
 
+3.6 Run command `npm run dev` to start your Web application and you should see the Web3Modal button loaded in your Web app page.
```
Comment (m4sterbunny) [inline docs/developing/integrate/wallets/integrating_walletconnect.mdx:127]: @mnedelchev-vn previously you used tabs to support yarn command -- pls extend same support here

File: docs/developing/integrate/wallets/integrating_walletconnect.mdx [hunk 2]
```diff
@@ -63,84 +65,81 @@ For this guide, we are configuring a React App.
...
 ### Step 4: Connect your wallet
+For this example we will be using [MathWallet](https://mathwallet.org/).
 
-Start your application: `npm/yarn start`
+4.1 After clicking the Web3Modal button you should be able to see several connecting options, select WalletConnect:
+<img src={buttonclicked} width="300" />
 
-You may now  connect your preferred wallet with Neon EVM Devnet within the WalletConnect interface.
+4.2 After selecting WalletConnect the popup will load a WalletConnect QR code:
+<img src={walletconnectqr} width="300" />
```
Comment (m4sterbunny) [inline docs/developing/integrate/wallets/integrating_walletconnect.mdx:136]: Can you cut your images closer to remove the grey background (or post-process it gone)?

**Review by m4sterbunny — CHANGES_REQUESTED** [review]

**Comment by m4sterbunny** [comment]
@mnedelchev-vn Pls review my commit that aligns with style, hopefully as you see more of these, the style will become 2nd nature -- if not -- [Google Developer Style Guide](https://developers.google.com/style) is your friend!

**Comment by mnedelchev-vn** [comment]
I'm ok with the changes, however I got 2 questions for you:
1. I saw you changed **DevNet** to **Devnet**, well then if you're not happy with this **DevNet** naming you should also contact whoever listed us in [Chainlist](https://chainlist.org/?search=Neon&testnets=true) to change the names also there?
2. I saw you've placed **an** before consonant, is this correct? Example - _For this guide, we are configuring an HTML Web App._

**Comment by m4sterbunny** [comment]
1. Yes [our style is Devnet](https://github.com/neonlabsorg/neon-evm.docs/blob/fb7782735f4adc4265e2123172239e2b9920105e/.github/styles/Grammarly.csv#L34) -- Please follow the Dali Lama and be the change you want == do find out how to get Chainlist to update to our style 
2. I wish English were easier on the non-natives, perhaps read this: https://www.writing-skills.com/hit-or-myth-use-an-before-h-words

File: docs/developing/integrate/wallets/integrating_walletconnect.mdx (hunk 2 above)
Comment (m4sterbunny) [reply docs/developing/integrate/wallets/integrating_walletconnect.mdx:143]: @mnedelchev-vn PTAL at this request

**Comment by m4sterbunny** [comment]
@mnedelchev-vn  there are some requested changes on the review that I conducted for you, please can you address those so that we can progress this PR. Thanks.

**Comment by mnedelchev-vn** [comment]
@m4sterbunny hey, I've added the requested changes, please check them.

File: docs/developing/integrate/wallets/integrating_walletconnect.mdx (hunk 2 above)
Comment (m4sterbunny) [reply docs/developing/integrate/wallets/integrating_walletconnect.mdx:143]: done

**Review by m4sterbunny — APPROVED** [review]

**Comment by m4sterbunny** [comment]
LGTM

**Pull request merged**